CMakeLists.txt

```diff
@@ -27,6 +27,16 @@ cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
 project(pocl)
 
+set(LATEST_KNOWN_CPP_VERSION "20")
+set(SUPPORTED_CPP_VERSION "11")
+
+option(ENABLE_LATEST_CPP "Upgrade CPP to ${LATEST_KNOWN_CPP_VERSION}. Required to get rid of unused variables warnings in non-gcc-compatible compilers. Can bring other benefits. Before a pull request build with this disabled." OFF)
```

Comment:
Also please use the same wording here CXX (in the help C++) and "standard version", not only "version".